app/controllers/project_roles_controller.rb

```diff
@@ -0,0 +1,46 @@
+class ProjectRolesController < ApplicationController
+  include CurrentProject
+  include ProjectLevelAuthorization
+
+  before_action only: [:create, :update] do
+    find_project(params[:project_id])
+  end
+
+  before_action :authorize_project_admin!, only: [:create, :update]
+
+  def index
+    render json: ProjectRole.all.map { |role| {id: role.id, display_name: role.display_name} }, root: false
```

Comment:
Added JSON serialiser to ProjectRole and removed this custom JSON from the controller.
Thanks for the heads up.